infra/nginx/nginx.conf

```diff
@@ -0,0 +1,35 @@
+user nginx;
+worker_processes  1;
+error_log stderr warn;
+
+events {
+    worker_connections  1024;
+}
+
+http {
+    include /etc/nginx/mime.types;
+    default_type application/octet-stream;
+    sendfile on;
+    keepalive_timeout 60;
+    gzip on;
+    client_max_body_size 100M;
+    log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
+                      '$status $body_bytes_sent "$http_referer" '
+                      '"$http_user_agent" "$http_x_forwarded_for"';
```

Comment:
Thoughts on logging format? Should we try for a specific structure for easier parsing?

Reply:
We should probably use json at some point. I just picked the default for now. https://stackoverflow.com/questions/25049667/how-to-generate-a-json-log-from-nginx#42564710

Reply:
Until we start ingesting logs into a log db, I'm going to leave this as non json.